packages/@react-spectrum/s2/src/SelectBoxGroup.tsx

```diff
@@ -0,0 +1,451 @@
+  /**
+   * The selection mode for the SelectBoxGroup.
+   * @default 'single'
+   */
+  selectionMode?: 'single' | 'multiple',
+  /**
+   * The currently selected keys in the collection (controlled).
+   */
+  selectedKeys?: Selection,
+  /**
+   * The initial selected keys in the collection (uncontrolled).
+   */
+  defaultSelectedKeys?: Selection,
+  /**
+   * Number of columns to display the SelectBox elements in.
+   * @default 2
+   */
+  numColumns?: number,
+  /**
+   * Gap between grid items.
+   * @default 'default'
+   */
+  gutterWidth?: 'default' | 'compact' | 'spacious',
+  /**
+   * Whether the SelectBoxGroup is disabled.
+   */
+  isDisabled?: boolean,
+  /**
+   * Whether to show selection checkboxes for all SelectBoxes.
+   * @default false
+   */
+  showCheckbox?: boolean
+}
+
+export interface SelectBoxProps extends StyleProps {
+  /**
+   * The value of the SelectBox.
+   */
+  value: string,
+  /**
+   * The label for the element.
+   */
+  children?: ReactNode,
+  /**
+   * Whether the SelectBox is disabled.
+   */
+  isDisabled?: boolean
+}
+
+interface SelectBoxContextValue {
+  allowMultiSelect?: boolean,
+  orientation?: Orientation,
+  isDisabled?: boolean,
+  showCheckbox?: boolean,
+  selectedKeys?: Selection,
+  onSelectionChange?: (keys: Selection) => void
+}
+
+export const SelectBoxContext = createContext<SelectBoxContextValue>({orientation: 'vertical'});
+export const SelectBoxGroupContext = createContext<ContextValue<Partial<SelectBoxGroupProps<any>>, DOMRefValue<HTMLDivElement>>>(null);
+
+const descriptionOnly = ':has([slot=description]):not(:has([slot=label]))';
+const labelOnly = ':has([slot=label]):not(:has([slot=description]))';
+const selectBoxStyles = style({
+  ...focusRing(),
+  outlineOffset: {
+    isFocusVisible: -2
+  },
+  display: 'grid',
+  gridAutoRows: '1fr',
+  position: 'relative',
+  font: 'ui',
+  boxSizing: 'border-box',
+  overflow: 'hidden',
+  width: {
+    default: 170,
+    orientation: {
+      horizontal: 368
+    }
+  },
+  height: {
+    default: 170,
+    orientation: {
+      horizontal: '100%'
+    }
+  },
+  minWidth: {
+    default: 144,
+    orientation: {
+      horizontal: 188
+    }
+  },
+  maxWidth: {
+    default: 170,
+    orientation: {
+      horizontal: 480
+    }
+  },
+  minHeight: {
+    default: 144,
+    orientation: {
+      horizontal: 80
+    }
+  },
+  maxHeight: {
+    default: 170,
+    orientation: {
+      horizontal: 240
+    }
+  },
+  padding: {
+    default: 24,
+    orientation: {
+      horizontal: 16
+    }
+  },
+  paddingStart: {
+    orientation: {
+      horizontal: 24
+    }
+  },
+  paddingEnd: {
+    orientation: {
+      horizontal: 32
+    }
+  },
+  gridTemplateAreas: {
+    orientation: {
+      vertical: [
+        'illustration',
+        '.',
+        'label'
+      ],
+      horizontal: {
+        default: [
+          'illustration . label',
+          'illustration . description'
+        ],
+        [descriptionOnly]: [
```

Comment:
I'm unaware of any description only use cases. I think we can drop this as an unsupported use case